### PR TITLE
Activate XML validation that should avoid issues like #809

### PR DESCRIPTION
**Pull request by juherr**

### CODE REVIEW

File: src/main/java/org/testng/xml/TestNGContentHandler.java [hunk 1]
```diff
@@ -501,6 +505,12 @@ public void xmlGroup(boolean start, Attributes attributes) throws SAXException {
    */
   @Override
   public void startElement(String uri, String localName, String qName, Attributes attributes) throws SAXException {
+    if (!m_validate && !m_hasWarn) {
+      Logger.getLogger(TestNGContentHandler.class).warn("It is heavy recommended to add " +
+              "\"<!DOCTYPE suite SYSTEM \"http://testng.org/testng-1.0.dtd\" >\" at the top of your file. " +
+              "Otherwise, TestNG may fail or not working as expected");
```
Comment (cbeust) [inline src/main/java/org/testng/xml/TestNGContentHandler.java]: "or not work"

**Comment by cbeust** [comment]
LGTM after the rephrasing.

File: src/main/java/org/testng/xml/TestNGContentHandler.java (hunk 1 above)
Comment (juherr) [inline src/main/java/org/testng/xml/TestNGContentHandler.java]: Thank. I'm aware English is something I need to improve ;)

**Comment by cbeust** [comment]
Thanks @juherr !

**Pull request merged**